src/app/components/pages/Post.jsx

```diff
@@ -89,13 +89,21 @@ class Post extends React.Component {
             }
         }
 
-        const replies = dis.get('replies').toJS();
+        let replies = dis.get('replies').toJS();
 
         let sort_order = 'trending';
         if( this.props.location && this.props.location.query.sort )
            sort_order = this.props.location.query.sort;
 
         sortComments( content, replies, sort_order );
+
+        // Don't render too many comments
+        const commentLimit = (global['process'] !== undefined) ? 500 : 1000
```

Comment:
Why different limits here?

Reply:
Reasoning was to avoid spending too much time server-side while still letting the browsers render more comments

Reply:
``` (global['process'] !== undefined) ```
is there possibly a more expressive way of saying "on the server side?" even if you just move this out into a helper function somewhere, many less expert js devs don't know about the environmental differences between node and the browser.